onnxruntime/core/framework/config_options.h

```diff
@@ -15,6 +15,11 @@ namespace onnxruntime {
  * Provides infrastructure to add/get config entries
  */
 struct ConfigOptions {
+  // Maximum key/value string lengths specified in
+  // core/session/onnxruntime_session_options_config_keys.h
+  static constexpr size_t kMaxKeyLength = 1024;
+  static constexpr size_t kMaxValueLength = 2048;
```

Comment:
maximum key length increased from 128 to 1024.
maximum value length stayed the same at 2048 (although onnxruntime_session_options_config_keys.h had an outdated value of 1024).